js/states/Boot.js

```diff
@@ -10,6 +10,13 @@ AngryCoders.BootState = {
         this.game.stage.backgroundColor = WhiteColor;
         //Set the scaling options
        //this.scale.scaleMode = Phaser.ScaleManager.SHOW_ALL;
+
```

Comment:
should have uncommented the line above. 
Please a little bit above and below the lines you are editing.